faker/providers/internet/it_IT/__init__.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+from __future__ import unicode_literals
+from .. import Provider as InternetProvider
+
+
+class Provider(InternetProvider):
+    safe_email_tlds = ('com', 'net', 'eu', 'it', 'it', 'it')
+    free_email_domains = (
+        'libero.it', 'libero.it', 'libero.it',
+        'tim.it',
+        'tin.it',
+        'alice.it',
+        'virgilio.it',
+        'tiscali.it',
+        'fastwebnet.it',
+        'vodafone.it',
+        'poste.it',
+        'gmail.com', 'gmail.com', 'gmail.com',
+        'outlook.com',
+        'live.com',
+        'hotmail.com',
+        'hotmail.it',
+        'yahoo.com',
+        'tele2.it '
```

Comment:
can you fix the trailing space on this string? while you're there, also add a courtesy trailing comma to the tuple :)